<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DIPL-5CG706063P-14" ignored="0" name="TiE Functional Regression Test - chrome" tests="3" failures="3" timestamp="2020-03-17T11:59:51 IST" time="55.168" errors="0">
  <testcase name="verifyAllSubMenuClickkableTest" time="0.115" classname="com.qa.tie.tests.InvoicePageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.tie.tests.InvoicePageTest.verifyAllSubMenuClickkableTest(InvoicePageTest.java:60)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- verifyAllSubMenuClickkableTest -->
  <testcase name="FilterInvoiceWithCustomerTest" time="0.006" classname="com.qa.tie.tests.InvoicePageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.tie.tests.InvoicePageTest.FilterInvoiceWithCustomerTest(InvoicePageTest.java:65)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- FilterInvoiceWithCustomerTest -->
  <testcase name="verifyAllHeaderTest" time="0.005" classname="com.qa.tie.tests.InvoicePageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.tie.tests.InvoicePageTest.verifyAllHeaderTest(InvoicePageTest.java:70)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- verifyAllHeaderTest -->
</testsuite> <!-- TiE Functional Regression Test - chrome -->
